vue-frontend/src/components/contact/NewContactForm.vue

```diff
@@ -0,0 +1,562 @@
+      return (
+        this.showNameValidationError ||
+        this.showEmailValidationError ||
+        this.showValidEmailError ||
+        this.showProjectInfoValidationError ||
+        this.showReferenceValidationError ||
+        this.showInvestValidationError
+      );
+    },
+    submitForm() {
+      if (!this.validateForm()) {
+        this.mixpanel.track("tap_contact_submit_button");
+        this.disableInput = true;
+        this.showLoader = true;
+        let formData = {
+          name: this.name,
+          email: this.email,
+          project_info: this.projectInfo ? this.projectInfo : "NA",
+          reference: this.reference,
+          invest: this.invest,
+          nda: this.NDA,
+          contact_type: "",
+        };
+        var head = document.getElementsByTagName("head")[0];
+        var script = document.createElement("script");
+        script.type = "text/javascript";
+        script.onload = () => {
+          // verify recaptcha
+          grecaptcha.enterprise.ready(() => {
+            grecaptcha.enterprise
+              .execute(import.meta.env.VITE_RECAPTCHA_SITE_KEY, {
+                action: "verify",
+              })
+              .then((token) => {
+                formData.token = token;
+                axios
+                  .post(config.API_BASE + "/api/send-contact-mail", formData)
+                  .then(() => {
+                    setTimeout(() => {
+                      this.showSuccessMessagePopup = true;
+                      this.showLoader = false;
+                    }, 1000);
+                  })
+                  .catch((err) => {
+                    if (err.response.status == 401) {
+                      this.errorMessage = "Invalid recaptcha score";
+                    }
+                    this.showErrorMessagePopup = true;
+                    this.showLoader = false;
+                  });
+              })
+              .catch(() => {
+                this.errorMessage = "Invalid recaptcha score";
+                this.showErrorMessagePopup = true;
+                this.showLoader = false;
+              });
+          });
+        };
+      }
+      script.src =
+        "https://www.google.com/recaptcha/enterprise.js?render=" +
+        import.meta.env.VITE_RECAPTCHA_SITE_KEY;
+      head.appendChild(script);
+    },
+    closePopUps(e) {
+      const showList = this.$refs["invest-list"];
+      if (showList && !showList.contains(e.target)) {
+        this.showList = false;
+        if (this.invest == "") {
+          this.floatable = false;
+        }
+      }
+    },
+    setOption(option) {
+      this.invest = option;
+      this.showInvestValidationError = this.invest.trim().length == 0;
+      this.showList = false;
+    },
+    toggleList() {
+      this.showList = !this.showList;
+      this.floatable = this.showList || this.invest.trim().length > 0;
+    },
+    toggleNDA() {
+      this.NDA = !this.NDA;
+    },
+    reloadWindow() {
```

Comment:
We don't need this method in the contact form, we need it in the home page footer to scroll to the top after submitting the form